eth/api.go

```diff
@@ -450,12 +452,113 @@ func formatError(err error) string {
 	return err.Error()
 }
 
+// jsLogCollector provides an implementation of StructLogCollector that 
+// evaluates a Javascript function for each VM execution step.
+type jsLogCollector struct {
+	traceobj *otto.Object
+	err error
+}
+
+
+// fakeBig is used to provide an interface to Javascript for 'big.NewInt'
+type fakeBig struct { }
+
+// NewInt creates a new big.Int with the specified int64 value.
+func (self *fakeBig) NewInt(x int64) *big.Int {
```

Comment:
Please don't use `self`, as it's really bad practice in Goland :) The more advanced linters whine quite verbosely about it. It's usually a single letter or a few letter word if possible. Here's I'd use `fb` myself :)